run-codemods.js

```diff
@@ -0,0 +1,102 @@
+#!/bin/env node
```

Comment:
On macOS, env is in /usr/bin/env. Since the instructions explicitly include node on the command line, that's not a show-stopper, but I wanted to point it out.

Reply:
I will switch to `/usr/bin/env` since that's more cross platform as you point out.
I just was trying to be consistent with the original `/bin/zsh`